src/helpers/config.js

```diff
@@ -0,0 +1,3 @@
+export const config = {
+  POINT_COUNT: 3
+};
```

Comment:
```suggestion
export const Config = {
  POINT_COUNT: 3
};
```

Ну и название Config такое себе. Конфиг чего?))